config/dogstatsd.yml

```diff
@@ -0,0 +1,12 @@
+production:
+  host: brain.prod-us-a.clio.systems
+  port: 8125
+production-ca:
+  host: brain.prod-ca-a.clio.systems
+  port: 8125
+production-eu:
+  host: 10.0.1.250
+  port: 8125
+staging:
+  host: 10.255.0.30
+  port: 8125
```

Comment:
This file has to be present for the tests to work. The contents don't matter; I just copied it over from Manage.